controlplane/kubeadm/internal/workload_cluster_coredns_test.go

```diff
@@ -287,62 +291,75 @@ kind: ClusterConfiguration
 	// Namespace/Name (e.g. The CoderDNS ConfigMap & Deployment, the kubeadm ConfigMap), it is not possible to run the use cases in parallel.
 	for _, tt := range tests {
 		g := NewWithT(t)
-		t.Log(tt.name)
-
-		for _, o := range tt.objs {
-			// NB. deep copy test object so changes applied during a test does not affect other tests.
-			o := o.DeepCopyObject().(client.Object)
-			g.Expect(testEnv.Create(ctx, o)).To(Succeed())
-		}
 
-		w := &Workload{
-			Client:          testEnv.GetClient(),
-			CoreDNSMigrator: tt.migrator,
-		}
-		err := w.UpdateCoreDNS(ctx, tt.kcp)
-		if tt.expectErr {
-			g.Expect(err).To(HaveOccurred())
-			return
-		}
-		g.Expect(err).ToNot(HaveOccurred())
+		t.Run(tt.name, func(t *testing.T) {
```

Comment:
Does running test in parallel creates problems, given that all the tests insist on the same CoreDNS ConfigMap & Deployment and on the same kubeadm ConfigMap?

Reply:
This was added to attach a cleanup to each of the subtest, and not for running the tests in parallel.

Reply:
ok with that given that it seems this is not introducing problems